src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -810,19 +810,22 @@ public function testTimeoutWithActiveConcurrentStream()
         }
     }
 
+    /**
+     * @group transient-on-macos
+     */
     public function testTimeoutOnInitialize()
     {
         $p1 = TestHttpServer::start(8067);
-        $p2 = TestHttpServer::start(8078);
+        $p2 = TestHttpServer::start(8077);
```

Comment:
why changing the port ?

Reply:
reverting #44636 which apparently doesn't work